lua-tree/share/lua/5.1/kong/tools/utils.lua

```diff
@@ -26,8 +26,7 @@ local fmt           = string.format
 local find          = string.find
 local gsub          = string.gsub
 local split         = pl_stringx.split
-local re_find       = require "go.re2".find
```

Comment:
Weird that this is even here, since this removal has been in the patch for months. Nothing to do here.

Reply:
That's why I wanted to clean it up. I'll leave it in per your suggestion.

Reply:
Actually, I went back and looked at both find and match signatures between gluare and ngx.re. They both methods are not an exact match. I've changed this line to instead reference the original ngx.re.find (even though it is still unused). This satisfies the goal of removing the need to patch custom plugins.

Reply:
nope they are not a 1:1 either unfortunately; we have had to make changes to some of the regex formats used.